sharepoint/sharepoint-ps/sharepoint-online/Set-SPOTenant.md

```diff
@@ -278,6 +279,18 @@ PS > Set-SPOTenant -SharingDomainRestrictionMode "BlockList" -SharingBlockedDoma
 
 This example enables users to share with all external collaborators except for those on the BlockedDomainList.
 
+### EXAMPLE 16
+
+```powershell
+PS > Set-SPOTenant -EnableVersionExpirationSetting $true
+```
+
+This example opts your tenant into public preview of Improved Version History controls feature. This feature is being tracked on the Microsoft 365 Public Roadmap under ID [145802](https://www.microsoft.com/en-us/microsoft-365/roadmap?filters=&searchterms=145802).
```

Comment:
>Improved Version History [](http://example.com/codeflow?start=53&length=24)

this is OK as Title Case since it's descripting the preview feature name

Reply:
Ok, I will keep it.